oracle/oracle-how-to/use-bastion-to-access-VM.rst

```diff
@@ -91,45 +137,155 @@ When the bastion plugin shows up as *Running* in the cloud console, you should s
 
-* Session type: Managed SSH session
-* Session name: any name of your choice
-* Username: ubuntu  (if your VM is an Ubuntu instance, the default user is ubuntu)
-* Compute instance: <the VM that you would like to access through the bastion>
-* Add SSH key: Add a public SSH key to inject into the bastion (you must have access to the private key so you can access it)
+Once the bastion goes into an *Active* state, you can create a session using either the console or the CLI.
+
+.. tabs::
+
+    .. group-tab:: Using console
+
+        Select your bastion and choose :guilabel:`Create session`. Fill in the details:
+
+        * Session type: Managed SSH session
+        * Session name: any name of your choice
+        * Username: ubuntu  (if your VM is an Ubuntu instance, the default user is ubuntu)
+        * Compute instance: <the VM that you would like to access through the bastion>
+        * Add SSH key: Add a public SSH key to inject into the bastion and the VM (you must have access to the corresponding private key)
+
+        .. image:: use-bastion-to-access-VM-images/4_create_session.png
+
+    .. group-tab:: Using CLI
+
+        Run:
+
+        .. code::
+
+            oci bastion session create-managed-ssh \
+                    --bastion-id <bastion_ocid> \
+                    --ssh-public-key-file <path_to_id_rsa.pub_key> \
+                    --target-resource-id <virtual_machine_ocid> \
+                    --target-os-username ubuntu
+        
+        where
+
+        * ``<path_to_id_rsa.pub_key>`` should be replaced with a public SSH key that you want injected into the bastion and the VM (you must have access to the corresponding private key)
+        * ``<virtual_machine_ocid>`` should be replaced with the OCID of the VM to be accessed and 
+        * 'ubuntu' should be replaced with the username that you would like to use to SSH into your VM ('ubuntu' is the default username in Ubuntu images)
 
-.. image:: use-bastion-to-access-VM-images/4_create_session.png
+.. note::
 
+    The SSH key that you use here, will be temporarily added to ``.ssh/authorized_keys`` in both the bastion and the VM. So when you try to access the VM in the next step, you'll have to provide your private key twice -  once to SSH into the bastion, and then again within a proxy command to SSH into the VM.
 
 
 Access the VM
 -------------
 
-When the session is created, use the *Copy SSH command* from the 3 dots menu to get a command similar to:
+You can access the VM using either the console or the CLI.
+
+.. tabs::
+
+    .. group-tab:: Using console
+
+        When the session is created, use the *Copy SSH command* from the 3 dots menu to get a command similar to:
+
+        .. code::
+
+            ssh -i <privateKey> -o ProxyCommand="ssh -i <privateKey> -W %h:%p -p 22 ocid1.bastionsession.oc1.sa-saopaulo-1.amaaaaaaniwq6sya34ss2bksayws7o4lovrtmvxg3e3baqb64vxw7nz447kq@host.bastion.sa-saopaulo-1.oci.oraclecloud.com" -p 22 ubuntu@10.0.0.86
```

Comment:
Just double checking that this long alphanumeric string is something we want to expose in the docs; I don't have enough context to know what it refers to, if it should be secret, or if it's likely to be unstable

Reply:
We do mention that the user will see something **similar** to it.
@fabiomirmar or @hammerstefan wdyt? Is it problematic? Should we change it?

Reply:
This one seems better, I'll replace it. Apart from security reasons it also shorter and easier on the eye!